src/styles/index.scss

```diff
@@ -2,2 +1,44 @@
+@import './header';
+@import './cards';
+@import './stars';
+
+* {
+  box-sizing: border-box;
+  display: block;
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  max-width: 1024px;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
+  display: grid;
+
+  --column-count: 1;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    &__container {
+      --column-count: 2;
+    }
+  }
+
+  @media (min-width: 768px) {
+    &__container {
+      --column-count: 3;
+    }
```

Comment:
Similar to the previous comment, you can directly set `--column-count: 3;` without using `&__container`.